test/integration/targets/setup_paramiko/install-FreeBSD-13-python-2.yml

```diff
@@ -0,0 +1,6 @@
+- name: Setup remote constraints
+  include_tasks: setup-remote-constraints.yml
+- name: Install Paramiko for Python 2 on FreeBSD 13
+  pip: # no package in pkg, just use pip
+    name: paramiko
+    extra_args: "-c {{ remote_constraints }}"
```

Comment:
Do we want the test fixes in a separate PR? That's helpful if we want to backport these test fixes separate from the addition of FreeBSD 13.

Reply:
These changes are only useful if the addition of FreeBSD 13 is backported anyway. This setup role basically just has a bunch of distro-and-version-specific file includes, so when we bump FreeBSD, we need to update it. It's been that way the [last](https://github.com/ansible/ansible/commit/4357a78130d84785720750d3d6801db5cf4df464) few [times](https://github.com/ansible/ansible/commit/11b7091c84ed6cf9576f319118b88f2a81894764) I've updated freebsd as well.